gpcontrib/arenadata_toolkit/sql/arenadata_toolkit_tracking.sql

```diff
@@ -114,6 +128,14 @@ FROM arenadata_toolkit.tables_track;
 
 DROP TABLE arenadata_toolkit.tracking_t1;
 
+-- Set empty relkinds. The track result set should be empty.
+SELECT arenadata_toolkit.tracking_set_relkinds('');
```

Comment:
Such request 
```
postgres=# SELECT arenadata_toolkit.tracking_set_relkinds(NULL);
```
results in 
```
server closed the connection unexpectedly
	This probably means the server terminated abnormally
	before or while processing the request.
The connection to the server was lost. Attempting reset: Failed.
!> 
```
on my side. I think we need to improve input check here.

The same for `tracking_register_schema()`, `tracking_set_relstorages()`.